test/integration/full/strict-csp/strict-csp.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf8" />
+		<script id="check-phantomjs">
+			// phantomjs requires running in non-strict csp
+			if (!window.PHANTOMJS) {
+				var thisScript = document.getElementById('check-phantomjs');
+				var csp = document.createElement('div');
```

Comment:
Why not `var meta = document.createElement('meta')`?

Reply:
Was just being lazy to not have to add 2 different attributes through `createAttribute` :)